srcpkgs/devilution/INSTALL.msg

```diff
@@ -0,0 +1,5 @@
+This package does NOT include the Diablo I game data, which are required to
+play the game.
+
+See the DevilutionX FAQ for more information:
+  https://github.com/diasurgical/devilutionX#faq
```

Comment:
No.

INSTALL.msg is not recommended.
Maybe, at the absolute most, a single line:
`This package does NOT include the required Diablo I game data`

Reply:
I used this one as a [reference](srcpkgs/dhewm3/INSTALL.msg), as the game has no manual man, I believe the faq link, it is necessary for configuration guidelines and game data.

Reply:
>  Maybe, at the absolute most, a single line:
> `This package does NOT include the required Diablo I game data`

Reply:
Removed lines.